tsconfig.json

```diff
@@ -10,12 +10,15 @@
     "paths": {
       "@vueuse/core": ["./packages/core/index.ts"],
       "@vueuse/core/*": ["./packages/core/*"],
-      "@vueuse/docs-utils": ["./packages/.vitepress/plugins/utils.ts"],
       "@vueuse/integrations": ["./packages/integrations/index.ts"],
       "@vueuse/metadata": ["./packages/metadata/index.ts"],
       "@vueuse/math": ["./packages/math/index.ts"],
       "@vueuse/shared": ["./packages/shared/index.ts"],
-      "@vueuse/shared/*": ["./packages/shared/*"]
+      "@vueuse/shared/*": ["./packages/shared/*"],
+      "@vueuse/components": ["./packages/components/index.ts"],
+      "@vueuse/components/*": ["./packages/components/*"],
+      "@vueuse/rxjs": ["./packages/rxjs/index.ts"],
+      "@vueuse/rxjs/*": ["./packages/rxjs/*"]
```

Comment:
@antfu the typecheck of the components and rxjs imports needs this to be built. 

Previously the demos imported from relative paths, so we never needed this. But maybe we dont want to add this 🤔